ddtrace/internal/compat.py

```diff
@@ -275,6 +275,23 @@ def maybe_stringify(obj):
     return None
 
 
+def to_bytes_py2(n, length=1, byteorder="big", signed=False):
```

Comment:
nit: can we change these to positional arguments? the defaults aren't used anywhere and could lead to misusage.

also the `signed` argument is not used.

(unless this function is just implementing the arguments/interface of the `to_bytes` in python 3. But then the `signed` argument isn't used at all so this is just a partial implementation which could lead to misusage if someone thinks it works)

Reply:
Yep. Only used in one place so it's no problem to change this and remove the extra argument.